app/assets/landingpage/app.css

```diff
@@ -638,6 +638,10 @@ section#intro {
     margin-left: 1.5em;
     margin-right: 1.5em; } }
 
+section#whatisgitcoin .actions {
+  display: none
+}
```

Comment:
let's remove this and use `d-none` / `d-md-none` / etc

Reply:
i tried this... but didn't work as required. @thelostone-mc